packages/qwik/src/core/render/jsx/types/jsx-qwik-events.ts

```diff
@@ -12,6 +12,29 @@ interface BaseSyntheticEvent<E = object, C = any, T = any> {
   stopPropagation(): void;
   isPropagationStopped(): boolean;
   persist(): void;
+  /**
+   * @deprecated Qwik does not allow `preventDefault()`.
+   * Add a `preventdefault:{event}` attribute to the element instead.
+   *
+   * @see https://qwik.builder.io/docs/components/events/#prevent-default
+   *
+   * @example
+   * ```tsx
+   * <a
+   *  href="/about"
+   *  preventdefault:click // This will prevent the default behavior of the "click" event.
+   *  onClick$={(event) => {
+   *    // event.preventDefault() would not work here
+   *    // because of Qwik's async nature
+   *    trackAnalyticsEvent(event.currentTarget)
```

Comment:
you mean `event.target`?

Reply:
yeah, i think event.currentTarget is not a good idea in Qwik either, since we capture the event, it's likely to be incorrect. I would rather use `event.target` or the second argument of the callback:

```
onClick$={(event, currentEvent) => {
```

Maybe we can set event.currentEvent as deprecated as well, and recommend the alternative

Reply:
sounds good, have updated in latest commit https://github.com/BuilderIO/qwik/pull/1857/commits/fcdbd53458f421cb766acf7a869da06be073f125

preview: 
![image](https://user-images.githubusercontent.com/18399089/197586253-bd3cf6a6-bb6c-41aa-ae1c-0a901d59f0a9.png)